frontend/src/components/FundsPullPreApproval/RejectModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Button, Modal, ModalBody } from "reactstrap";
+import CloseButton from "../CloseButton";
+import { Approval } from "../../interfaces/approval";
+import BackendClient from "../../services/backendClient";
+
+interface RejectModalProps {
+  approval: Approval | undefined;
+  open: boolean;
+  onClose: () => void;
+}
+
+function RejectModal({ approval, open, onClose }: RejectModalProps) {
+  const updateApproval = async () => {
+    try {
+      await new BackendClient().updateApprovalStatus(
+        approval!.funds_pull_pre_approval_id,
+        "rejected"
+      );
+    } catch (e) {
+      console.error(e);
+    }
+
+    onClose();
+  };
+
+  return (
+    <Modal className="modal-dialog-centered" isOpen={open} onClosed={onClose}>
```

Comment:
You should have a property `centered` for that purpose

Reply:
@vladkucherov - centered open the modal in the upper part of the screen we want him in the center

Reply:
sounds like a bug